File: python/MASTER_SLAVE_WHOLE_PART_ABSTRACT_FACTORY/MASTER_SLAVE_WHOLE_PART_ABSTRACT_FACTORY/whole_part/ReservacionSencilla.py
from .Reservacion import Reservacion

class ReservacionSencilla(Reservacion):
    
    def __init__(self, datosCliente, datosCompra, datosFuncion):
        self.datosCliente = datosCliente
        self.datosCompra = datosCompra
        self.datosFuncion = datosFuncion

File: php/MASTER_SLAVE_WHOLE_PART_ABSTRACT_FACTORY/reservaciones/ReservacionSencilla.php
<?php

require './reservaciones/Reservacion.php';

/**
 * Description of ReservacionNormal
 *
 * @author giltenorio
 */
class ReservacionSencilla extends Reservacion{
    //put your code here
    
    
    public function __construct($datosCliente, $datosCompra, $datosFuncion) {
         parent::__construct($datosCliente, $datosCompra, $datosFuncion);
    }
    
    public function gestionarReservacion(){
        echo 'Gestionar Reservación sencilla';
    }
    
    public function mostrarReservacion(){
        return parent::mostrarReservacion();
    }
    
    

}
